app/javascript/spotlight/admin/blocks/resources_block.js

```diff
@@ -59,7 +60,16 @@ Core.Block.Resources = (function(){
           .map(word => word.charAt(0).toUpperCase() + word.slice(1))
           .join('');
     },
-
+    _itemSelectImageLink: function(block_item_id, doc_id, index) {
+      // If image selection is not possible for this block, then do not show
+      // image selection link
+      if (!this.show_image_selection) return ``;
+      var url = $('form[data-exhibit-path]').data('exhibit-path') + '/select_image?';
```

Comment:
```suggestion
      var url = $('form[data-exhibit-path]').data('exhibit-path') + '/select_image?';
      var url = document.querySelector('form[data-exhibit-path]').dataset.exhibitPath + '/select_image?'
```

Reply:
I started down the path of changing things to (non JQuery) JavaScript, but objects are passed back and forth between modules/functions, and what works for one does not work for the other.  I've set up another branch to clean this up and use straight up JavaScript but I will plan on doing that in another PR.  FYI: After discussion with @dnoneill , this suggested approach seems fine.